www/profile/index.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+$servername = "localhost";
+$username = "worker";
+$password = "swiffcat";
+$db = "socialmedia";
+
+// Create connection
+$conn = new mysqli($servername, $username, $password, $db);
+
+// Check connection
+if (!$conn) {
+    die("Connection failed: " . mysqli_connect_error());
```

Comment:
This is not standard for this repo

Reply:
I saw it used in feed and search\posts so I thought it was alright.
Should I use
if ($conn->connect_error) {
        header("HTTP/1.1 500 Internal Server Error");
        die();
    }
instead?

Reply:
Yes